wiki/Beatmap/Marathon/fr.md

```diff
@@ -10,8 +10,8 @@ Le terme **Marathon** est utilisé pour décrire les [beatmaps](/wiki/Beatmap) d
 
 Lors de l'introduction des beatmaps marathons, elles devaient à l'origine durer au moins *six* minutes et ne pouvaient inclure qu'une seule difficulté (à l'exception du fait que chaque beatmap pouvait également inclure une seule difficulté [osu!taiko](/wiki/Game_mode/osu!taiko)).
 
-Les beatmaps marathon devaient également avoir leur difficulté unique nommée "Marathon" pour contourner le système de soumission des beatmaps marathon de l'époque et empêcher les beatmaps marathon d'entrer dans la catégorie [en attente](/wiki/Beatmap/Category#work-in-progress-et-en-attente). Lors du classement, les beatmaps marathon entrent dans la catégorie "Approuvée" au lieu de la catégorie [classée](/wiki/Beatmap/Category#classée) pour signaler qu'il s'agit d'une beatmap marathon. Cette catégorie a ensuite été abandonnée au profit de la catégorie classée.
+Les beatmaps marathon devaient également avoir leur difficulté unique nommée "Marathon" pour contourner le système de soumission des beatmaps marathon de l'époque et empêcher les beatmaps marathon d'entrer dans la catégorie [en attente](/wiki/Beatmap/Category#work-in-progress-et-en-attente). Lors du classement, les beatmaps marathon entrent dans la catégorie [approuvée](/wiki/Beatmap/Category#approuvée) au lieu de la catégorie [classée](/wiki/Beatmap/Category#classée) pour signaler qu'il s'agit d'une beatmap marathon. Cette catégorie a ensuite été abandonnée au profit de la catégorie classée.
 
-Les beatmaps marathon ont ensuite été modifiées pour inclure des beatmaps avec des drain de la vie de cinq minutes et permettant de nommer des difficultés personnalisées. Cela a duré jusqu'à ce que des règles de propagation plus indulgentes soient introduites en 2019.
+Les beatmaps marathon ont ensuite été modifiées pour inclure des beatmaps avec [des drain de la vie](/wiki/Gameplay/Drain_time) de cinq minutes et permettant [de nommer des difficultés personnalisées](/wiki/Ranking_Criteria/Difficulty_Naming). Cela a duré jusqu'à ce que des règles de propagation plus indulgentes soient introduites en 2019.
```

Comment:
```suggestion
Les beatmaps marathon ont ensuite été modifiées pour inclure des beatmaps avec [des drains de vie](/wiki/Gameplay/Drain_time) de cinq minutes et permettant [de nommer des difficultés personnalisées](/wiki/Ranking_Criteria/Difficulty_Naming). Cela a duré jusqu'à ce que des règles de propagation plus indulgentes soient introduites en 2019.
```

Reply:
"drain de vie" will probably be changed in the future by "Drain Time" because it make more sens but the fr page for this term need to be updated too, so it will be made in another pr later